Formula/aamath.rb

```diff
@@ -15,6 +15,11 @@ class Aamath < Formula
     sha256 "0212e0b5844ea1a491bc7d4fcab2b590921042b28bc50e79c36cd9e15d08e2aa" => :mavericks
   end
 
+  unless OS.mac?
+    depends_on "bison" => :build # for yacc
+    depends_on "flex" => :build
```

Comment:
```suggestion
    uses_from_macos "flex" => :build
```
Does `uses_from_macos "…" => :build` work?

Reply:
It does! I'm never sure which dependencies are macOS-provided until I have time to search Homebrew/homebrew-core for the `uses_from_macos` equivalent.